ogr/ogrsf_frmts/gml/ogrgmldatasource.cpp

```diff
@@ -28,10 +28,12 @@
 #include "cpl_http.h"
 #include "cpl_string.h"
 #include "cpl_vsi_error.h"
+#include "cpl_json.h"
```

Comment:
probably unneeded since the JSON parsing is now fully dealt with in ogr_schema_override.h
```suggestion
```